app/javascript/lib/scp-api.js

```diff
@@ -120,6 +120,67 @@ export function setMockOrigin(origin) {
   mockOrigin = origin
 }
 
+/** Constructs and encodes a URL parameter, if it has a value */
+function encodeParam(param, value, isFirst=false) {
+  const delimiter = isFirst ? '?' : '&'
+  if (!value || value === '') {
+    // Omit empty parameters in API request
+    return ''
+  } else {
+    return `${delimiter}${param}=${encodeURIComponent(value)}`
+  }
+}
+
+/**
+* Returns initial content for the "Explore" tab in Study Overview
+* @param {String} studyAccession Study accession
+*/
+export async function fetchExploreInitialization(studyAccession, mock=false) {
```

Comment:
I'm not sure I understand 'initialization' in this function name.  I would mirror the api name more closely, so that we're not tying scp-api.js function names to specific UI elements.  "fetchStudyExploreVizData" or somesuch

Reply:
I went with `fetchExplore` to concisely mirror the API name; see b24e07f.